app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all
```

Comment:
This isn't quite right since this is all the ingredients, not the ingredients for a given chef. Since you have this route nested, you should have access to the chefs_id here, so you'd want to find the chef, then find all the ingredients that this chef has used, and those are then the ingredients we want to display.